config/config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Prefer env variables, use [dotenv](https://www.npmjs.com/package/dotenv)
Then you can create a .env file for your own variables and we do not need them commit ;-)

Reply:
This also ensures we can easily migrate to docker